docs/guides/token_guide.md

```diff
@@ -31,3 +27,125 @@ Below is an overview of how the migration will take place:
+
+
+### Migration Guide via Seed
+The new recovery phrase and the migration address will be handled by the wallet library.
+
+Our IOTA Wallet Firefly is basically using wallet rs for the migration. There are three main methods exposed from wallet.rs:
+- [getMigrationData](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L203)
+- [createMigrationBundle](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L215)
+- [sendMigrationBundle](https://github.com/iotaledger/firefly/blob/develop/packages/backend/bindings/node/index.ts#L219)
+
+#### Fetch migration data by a seed
+First, with a user provided legacy seed, we fetch migration data using getMigrationData(). It returns to us with the addresses and it's balance. It also returns address metadata such as if an address is spent. To be precise, [this](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/typings/migration.ts#L12-L16) is what getMigrationData() returns.
+
+#### Create a Migration Bundle
+Then, for the addresses we receive, we basically split them into chunks / bundles. We could create a single bundle for all addresses and migrate them, but we want to avoid that and cap the number of address a single bundle can contain, which happens [here](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/migration.ts#L279-L300). Note that for spent addresses, we only have a single input per bundle.
+
+#### Check for spent addresses
+The next step is to check if there are spent addresses on a user's seed. If there are, we bundle mine those addresses. This is done through createMigrationBundle by setting the third param mine to true. For the rest of the bundles, we will keep this param to false. createMigrationBundle will return us the migration bundle.
+
+#### Send them to the new network
+The last step is to simply use sendMigrationBundle and broadcast them to the network.
+
+Another important point is how we select inputs. You can see how Firefly accomplishes this [here](https://github.com/iotaledger/firefly/blob/develop/packages/shared/lib/migration.ts#L228-L277).
+
+
+Basically, we need to make sure we create bundles that have accumulative balance >= 1Mi.
+
+#### Migration Code Example with Node.js
+This example creates a new database, account, and migrates funds from the legacy network to the chrysalis network.
+
+ 
+```javascript=
+require('dotenv').config()
+
+async function run() {
+    const { AccountManager, SignerType } = require('@iota/wallet')
+    const manager = new AccountManager({
+        storagePath: './migration-database',
+    })
+    manager.setStrongholdPassword(process.env.SH_PASSWORD)
+    manager.storeMnemonic(SignerType.Stronghold)
+
+    const account = await manager.createAccount({
+        clientOptions: { node: "https://api.lb-0.testnet.chrysalis2.com", localPow: true },
```

Comment:
Guys, just to be sure. There should be an adddress to mainnet at the end of the day, right? The same goes to `https://nodes.devnet.iota.org`?

Reply:
You're right, the address should probably be `https:/nodes.iota.org` for the legacy network and `https://chrysalis-nodes.iota.org` for the new one and for the client options we also need to provide a custom network name `network: "chrysalis-mainnet"` as long as the new nodes aren't up, otherwise wallet.rs sends a request to the node and will return an error

Reply:
Good catch!